Assets/MixedRealityToolkit/_Core/Definitions/InputSystem/MixedRealityInputActionsProfile.cs

```diff
@@ -11,6 +12,34 @@ namespace Microsoft.MixedReality.Toolkit.Internal.Definitions.InputSystem
     [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Mixed Reality Input Actions Profile", fileName = "MixedRealityInputActionsProfile", order = 1)]
     public class MixedRealityInputActionsProfile : ScriptableObject
     {
+        private readonly string[] inputActionsMRTKDefaults = 
+        {
+            "Select",
+            "Menu",
+            "Grip",
+            "Pointer",
+            "Walk",
+            "Look",
+            "Interact",
+            "Pickup",
+            "Inventory",
```

Comment:
A few of these I'm not sure fits all use cases.

I thin this should def be something we leave open for devs to setup on their own.